acceptance/openstack/baremetal/noauth/ports_test.go

```diff
@@ -0,0 +1,71 @@
+package noauth
```

Comment:
There needs to be the following two lines before the `package` statement:

```
// +build acceptance baremetal ports
                                    
```

The newline is required.

Reply:
Ack